content/acknowledgements.en.md

```diff
@@ -52,7 +54,6 @@ _Grácias a todas las personas que contribuyeron al contenido de esta guía, aqu
   Liili Abuladze, 
```

Comment:
We should remove Pao Corrales from this list because she have to be on the Global Team list.

Reply:
Done!